@navikt/navno/react/src/card/base-card/sizes/LargeCard.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { forwardRef } from "react";
+import cl from "classnames";
+import { OverridableComponent, Link } from "@navikt/ds-react";
+import { Heading, BodyLong, BodyShort } from "@navikt/ds-react";
+import { Animation } from "../../../animation";
+import { useInteractions } from "../useInteraction";
+
+export interface LargeCardProps
+  extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  activeAnimation?: any;
+  category: string;
+  hoverAnimation?: any;
+  text: string;
+  title: string;
+  type?: "situation" | "product" | "tool" | "general";
+}
+
+const LargeCard: OverridableComponent<
+  LargeCardProps,
+  HTMLAnchorElement
+> = forwardRef(
+  (
+    {
+      as: Component = "article",
```

Comment:
Skulle dette vært `as: Component = "a"`, siden props extender `AnchorHTMLAttributes`?

Reply:
Ah, den extender feil ja. Hadde en diskusjon med UU på kortene + litt egenlæring: Kortene bør ikke wrappes i en anchor fordi skjermleseren da leser opp både lenke og tekst. Istedet bør kortet være en <article> og så bør selve tittelen være en lenke.

Jeg tenker derfor at riktig extending er React.HTMLAttributes<HTMLElement>. Finner ingen egen type for article-elementet egentlig...